app/backend/models/trading-eq-follow.js

```diff
@@ -0,0 +1,17 @@
+const {mongoose} =  require('../db');  // The mongodb connector library
+
+let TradingEquipmentFollow = mongoose.model('TradingEquipmentFollow', {
+  UserId: {
+    type: String,
```

Comment:
UserId is supposed to be of type ObjectId here.

Reply:
no need for ObjectID